src/linear_algebra/projective_space/subspace.lean

```diff
@@ -127,6 +121,26 @@ instance : complete_lattice (subspace K V) :=
 instance subspace_inhabited : inhabited (subspace K V) :=
 { default := ⊤ }
 
+/-- The span of a set of points is contained in a subspace if and only if the set of points is
+contained in the subspace. -/
+lemma span_le_subspace_iff {S : set (ℙ K V)} {W : subspace K V} : span S ≤ W ↔ S ⊆ W :=
+gi.gc S W
+
+/-- If a set of points is a subset of another set of points, then its span will be contained in the
+span of that set. -/
+lemma span_mono {S T : set (ℙ K V)} (h : S ⊆ T) : span S ≤ span T :=
+galois_connection.monotone_l gi.gc h
+
+/-- The supremum of two subspaces is equal to the span of their union. -/
+lemma sup_eq_span_union (W S : subspace K V) : W ⊔ S = span (W ∪ S) :=
+by { apply symm, exact (galois_insertion.l_sup_u gi W S) }
+
+/-- The supremum of a collection of subspaces is equal to the span of the union of the
+collection. -/
+lemma Sup_eq_span_union (S : set (subspace K V)) :
+  Sup S = span ⋃ (W : subspace K V) (hW : W ∈ S), W :=
+by { apply symm, rw (Sup_eq_supr), exact galois_insertion.l_bsupr_u gi _ }
```

Comment:
Here I'd recommend:
```suggestion
lemma span_Union {ι} (s : ι → set (ℙ K V)) : span (⋃ i, s i) = ⨆ i, span (s i) :=
(@gi K V _ _ _).gc.l_supr
```
Note in particular that it's basically always better to work with an indexed family such as `s : ι → set (ℙ K V)` rather than a set of objects.